scripts/ableplayer-base.js

```diff
@@ -38,14 +43,15 @@
 var AblePlayerInstances = [];
 
 (function ($) {
-	$(document).ready(function () {
-
-		$('video, audio').each(function (index, element) {
-			if ($(element).data('able-player') !== undefined) {
-				AblePlayerInstances.push(new AblePlayer($(this),$(element)));
-			}
-		});
-	});
+	// Initialise all AblePlayerInstances - this can be called
+	// by the React component that imports ableplayer
+  window.initAllAblePlayers = function() {
```

Comment:
Why set this on window rather than exporting it?

Reply:
I *was* exporting it, before I decide to export `window.AblePlayer` instead. I think I'll remove the comment 😁

Reply:
I mean the value `window.initAllAblePlayers`. Why not move this code to the top level, and to `export const initAllAblePlayers = ...`? It's strange to have a module with exports but that also has some things that it sets on `window` and doesn't export.